ops/map_importer.py

```diff
@@ -170,6 +213,17 @@ def execute(self, context):
         self._object_instances = map_data['object_instances']
         self._object_data = map_data['object_data']
 
+        # Get all the col files from dff folder with the same region prefix as the current map section
+        if self.settings.load_collisions:
+            region_prefix = self.settings.map_sections[:-4].split('\\')[-1].split("_")[0].lower()
```

Comment:
We changed the back slashes to forward slashes, so this will need to be updated.

Reply:
Done.